econplayground/main/views.py

```diff
@@ -62,6 +62,30 @@ def test_func(self):
         return user_is_instructor(self.request.user)
 
 
+class CohortGraphCreateDetailView(
```

Comment:
I think I see what your intentions were here - was this to give each graph type a unique URL which can then be refreshed on? In any case - this new URL route seems to be unused in these current changes.

Reply:
Yes, I thought you had said to leave it in for now. I can remove it

Reply:
Alright - yeah let's go ahead and remove it for now. We can always add back in this functionality later if we find it's necessary.